examples/experimental/vz.yaml

```diff
@@ -11,3 +16,13 @@ mounts:
 - location: "/tmp/lima"
   writable: true
 mountType: "virtiofs"
+
+provision:
+- mode: system
+  # configure binfmt support for Rosetta
+  script: |
+    apt install binfmt-support
+    /usr/sbin/update-binfmts --install rosetta /mnt/lima-rosetta/rosetta \
+    --magic "\x7fELF\x02\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x3e\x00" \
+    --mask "\xff\xff\xff\xff\xff\xfe\xfe\x00\xff\xff\xff\xff\xff\xff\xff\xff\xfe\xff\xff\xff" \
+    --credentials yes --preserve no --fix-binary yes
```

Comment:
- No need to install binfmt-support if you write to `/proc/sys/fs/binfmt_misc` by yourself
- I'd expect this to be moved to a YAML property like `rosetta.binfmt: true`

Reply:
This was promising, but I'm hitting permissions issues as root. I suspect it's something to do with apparmor, and that the binfmt-support package probably adds a profile to allow this. Any ideas how I could work around this?

Reply:
Ah nvm I see, I have to use `/proc/sys/fs/binfmt_misc/register`